tests/services/inventory/test_data/mock_cai_resources.dump

```diff
@@ -34,7 +34,7 @@
 {"asset_type":"google.compute.InstanceGroup","name":"//compute.googleapis.com/projects/project1/regions/us-central1/instanceGroups/gke-cluster-1-default-pool-12345678-grp","resource":{"data":{"creationTimestamp":"2017-08-24T11:10:06.771-07:00","description":"This instance group is controlled by Regional Instance Group Manager 'gke-cluster-1-default-pool-12345678-grp'. To modify instances in this group, use the Regional Instance Group Manager API: https://cloud.google.com/compute/docs/reference/latest/instanceGroupManagers","fingerprint":"42WmSpB8rSM=","id":"1082","instance_urls":["https://www.googleapis.com/compute/v1/projects/project1/zones/us-central1-c/instances/ke_instance1","https://www.googleapis.com/compute/v1/projects/project1/zones/us-central1-c/instances/ke_instance2","https://www.googleapis.com/compute/v1/projects/project1/zones/us-central1-c/instances/ke_instance3"],"kind":"compute#instanceGroup","name":"gke-cluster-1-default-pool-12345678-grp","network":"https://www.googleapis.com/compute/v1/projects/project1/global/networks/default","region":"https://www.googleapis.com/compute/v1/projects/project1/regions/us-central1","selfLink":"https://www.googleapis.com/compute/v1/projects/project1/regions/us-central1/instanceGroups/gke-cluster-1-default-pool-12345678-grp","size":3,"subnetwork":"https://www.googleapis.com/compute/v1/projects/project1/regions/us-central1/subnetworks/default"},"parent":"//cloudresourcemanager.googleapis.com/projects/1041"}}
 {"asset_type":"google.compute.Firewall","name":"//compute.googleapis.com/projects/project2/global/firewalls/default-allow-rdp","resource":{"data":{"allowed":[{"IPProtocol":"tcp","ports":["3389"]}],"creationTimestamp":"2017-05-04T16:23:00.568-07:00","description":"Allow RDP from anywhere","direction":"INGRESS","id":"10722","kind":"compute#firewall","name":"default-allow-rdp","network":"https://www.googleapis.com/compute/beta/projects/project2/global/networks/default","priority":1000,"selfLink":"https://www.googleapis.com/compute/beta/projects/project2/global/firewalls/default-allow-rdp","sourceRanges":["0.0.0.0/0"]},"parent":"//cloudresourcemanager.googleapis.com/projects/1042"}}
-{"asset_type":"google.cloud.bigquery.Dataset","name":"//bigquery.googleapis.com/projects/1043/datasets/dataset1","resource":{"data":{"datasetReference":{"datasetId":"dataset1","projectId":"project3"},"location":"us","id":"project3:dataset1","kind":"bigquery#dataset"},"parent":"//cloudresourcemanager.googleapis.com/projects/1043"}}
```

Comment:
The location field is missing in mock_gcp_results.py it looks like.

Reply:
Updated.